### DISCUSSION TITLE
Add proxy and socks5 and tor feathers to Katana

### DISCUSSION THREAD

**Discussion by skalvin** [projectdiscovery/katana#246] [Ideas]
Hello, Developers

It's nice to Add proxy -proxy and socks5 -socks and tor -tor feathers to Katana tool

**Comment by MetzinAround** [comment]
@ehsandeep this user is looking for socks5 and tor in katana. Is this something we would consider in the future?

**Comment by ehsandeep** [comment]
```
   -proxy string  http/socks5 proxy to use
```

This is supported.
